### DISCUSSION TITLE
Feature request: Ignore "Block command line pushes that expose my email" for private repos

### DISCUSSION THREAD

**Discussion by jxu** [community/community#31263] [Code Security]
I know this option can simply be disabled, but it seems unnecessary to block email exposing pushes within private repos.

**Comment by swrobel** [comment]
Glad to see I'm not the only one who wants this! I want to block pushes to public repos that would expose my email, but I don't want github blocking my pushes to private repos where I've explicitly set a specific email for my commits there.
